### DISCUSSION TITLE
Open free shop items

### DISCUSSION THREAD

**Discussion by mercertom** [SergeyPotapov01/bot_Clash_Royale#14] [Ideas]
Can search for "free" button and open every free offer in the shop.

**Comment by SergeyPotapov01** [comment]
this is basically unrealistic.

to do this, you need to scroll through the clan chat to find this button

**Comment by mercertom** [comment]
Yes, a scroll loop is necessary for several clash Royale bot functions,
including this one. It's completely realistic. Multiple CR bots already do
it. Why do you think scrolling is unrealistic?
